src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Ball ball = new Lottery().getRandomBall();
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/197786371-2b5f8dfd-bbcb-4454-bc36-44e0526e8fd3.png)

Reply:
Let's discuss the requested changes. 
1. My task: "...in main() method, **create three balls** using getRandomBall()...". I have understood it as creating 3 exemplars of the class (perhaps I will need to use them further, due to the principle of scalability, theoretically).
2. For this purpose I use "Use for loop for creating several objects of the same class" from the checklist under the task. 
3. I understood, that implementing your request is possible (f.e., for educational purposes), but don't understand how it makes better the solution(
Would you explain it?

Reply:
dont write comments here, mentors wont see them until your next pull request - #jv_chat is made for discussions

3 it is not for educational purposes))
your task here is to make a method that prints 3 random balls, there wasn't mentioned any scalability
I wanted you to make a variable for Lottery, because you create a new instance of that object in each iteration - which was also mentioned in the checklist as a common mistake

also, you have to fix all the common mistakes. If mentors see any mistake from the checklist - they reject your solution immediately